packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -659,6 +659,32 @@ class WebViewController {
   Future<String> getTitle() {
     return _webViewPlatformController.getTitle();
   }
+
+  /// Set the scrolled position of this view.
```

Comment:
nit: empty line after the summary for all Dartdoc comments.

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
Of course, thx for pointing this out. Fixed in coming commit.